app/services/public_url_service.rb

```diff
@@ -18,7 +18,7 @@ def authenticate_url(address:)
     end
 
     def redirect_url(path:)
-      "#{website_root}/#{path}"
+      File.join(website_root, path)
```

Comment:
Won't this break on Microsoft Windows, or any OS that doesn't use `/` as the file separator?